addons/account/account.py

```diff
@@ -2168,6 +2168,13 @@ def _compute(self, cr, uid, taxes, price_unit, quantity, product=None, partner=N
                 total += r['amount']
         return res
 
+    def _map_included_tax(self, cr, uid, price, prod_taxes, line_taxes):
```

Comment:
For clarity I would also name the method something like `_fix_tax_included_price(...)` and put a small docstring to explain what it does:
> Subtract tax amount from price when corresponding "price included" taxes do not apply